src/main/java/it/gov/pagopa/payhub/activities/dto/treasury/FlussoTesoreriaPIIDTO.java

```diff
@@ -0,0 +1,25 @@
+package it.gov.pagopa.payhub.activities.dto.treasury;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class FlussoTesoreriaPIIDTO {
```

Comment:
The PII segregation is not demanded to the activity, but to the DAO!
Activity should not be aware of such persistence requirement.

ps: please align with @marcomatteuccieng to check if we will store all these info, treasury should not more have PII

Reply:
deleted FlussoTesoreriaPIIDTO